src/rime/dict/dict_compiler.cc

```diff
@@ -230,6 +231,26 @@ bool DictCompiler::BuildPrism(const string &schema_file,
         script.clear();
       }
     }
+
+#if 0
+    // build corrector
+    bool enable_correction = false; // Avoid if initializer to comfort compilers
+    if (config.GetBool("translator/enable_correction", &enable_correction) &&
+        enable_correction) {
+      boost::filesystem::path corrector_path(prism_->file_name());
+      corrector_path.replace_extension("");
+      corrector_path.replace_extension(".correction.bin");
```

Comment:
`".correct.bin"` which is shorter and matches `".reverse.bin"`. Yes, "correct" is an adjective.